### PR TITLE
Feat/open notes

### PR DESCRIPTION
**Pull request by Crash--**
- When we click on a Notes file, we call the stack to build the URL for the Notes app 
- If we click on a sharing link and this sharing only shared a Notes, let's do the same

### CODE REVIEW

File: src/drive/web/modules/drive/files.js [hunk 1]
```diff
@@ -24,13 +24,29 @@ export const isReferencedByAlbum = file => {
...
-  const myCozyUrl = client.getStackClient().uri
-  const noteCreatedOnCozy = get(file, 'cozyMetadata.createdOn')
-  return noteCreatedOnCozy.startsWith(myCozyUrl)
+export const fetchUrlToOpenANote = async (client, file) => {
+  const {
+    data: { note_id, subdomain, protocol, instance, sharecode, public_name }
+  } = await client
+    .getStackClient()
+    .collection('io.cozy.notes')
+    .fetchURL({ _id: file.id })
```
Comment (y-lohse) [inline src/drive/web/modules/drive/files.js]: No way to use `client.query` here?

File: src/drive/web/modules/navigation/FileExplorer.jsx [hunk 2]
```diff
@@ -187,32 +190,12 @@ export const handleFileOpen = (file, availableOffline, props, dispatch) => {
...
-    } else {
-      //If something went wrong previoulsy like the request to /apps has failled
-      //we display the note within the viewer
-      redirectToViewer()
+  const { client } = props
+  if (isNote) {
+    try {
+      window.location.href = await fetchUrlToOpenANote(client, file)
+    } catch (e) {
+      Alerter.error('alert.offline')
```
Comment (y-lohse) [inline src/drive/web/modules/navigation/FileExplorer.jsx]: We don't want to redirect to the viewer in this case?

Comment (Crash--) [reply src/drive/web/modules/navigation/FileExplorer.jsx]: Not yet, next PR. I've to update the Viewer to display a few lines on it. And for that I need to upgrade UI -> XP modal and so on :p

**Comment by Crash--** [comment]
> Cool! Testing the output of fetchUrlToOpenANote might be a good addition to this PR :) 

I'm currently adding this method to Cozy-Client and tests will be done there.

File: src/drive/web/modules/drive/files.js (hunk 1 above)
Comment (Crash--) [reply src/drive/web/modules/drive/files.js]: Will fix it in cozy-client, thanks :)

**Pull request merged**